src/http/get-ipr-000user-000repo-pull-000prEtc/check-form.js

```diff
@@ -21,8 +21,8 @@ const [
 if (!slug || !branch) {
 	throw 'args required: slug, branch';
 }
-if (!process.env.GH_TOKEN) {
-	throw 'GH_TOKEN env var required';
+if (!process.env.GITHUB_TOKEN) {
```

Comment:
does github provide both of these by default?

Reply:
All this is running in Lambda, so the token env var is whatever we want it to be. I observed the app was using both `GITHUB_TOKEN` + `GH_TOKEN`, so I figured we should standardize around one. It can be `GH_TOKEN` though, whatever works!